Networking/Networking/Remote/StripeRemote.swift

```diff
@@ -3,6 +3,19 @@ import Foundation
 /// Stripe (Extension): Remote Endpoints
 ///
 public class StripeRemote: Remote {
```

Comment:
I was going to comment that `StripeRemote` might be a bit misleading, as this does not attack Stripe but the Stripe Extension. But then I realized We have a WCPayRemote as well, and if WCPayRemote <==> WCPay Extension then StripeRemote <==> Stripe Extension. 

And yet, I commented anyway😊